cucumber-messages/ruby/messages.proto

```diff
@@ -1,7 +1,5 @@
 syntax = "proto3";
-package io.cucumber.messages;
```

Comment:
This file should not be modified. It is copied from `../messages.proto`. Change that file instead and run `source scripts/functions.sh && rsync_files`.

Changing the `package` value will break the Java implementation because it needs generated files to go into `io.cucumber.messages`. Maybe we can use `java_package` to set that specifically for Java - I haven't tried. See:
* https://developers.google.com/protocol-buffers/docs/proto3